skll/config.py

```diff
@@ -485,11 +485,20 @@ def _parse_config_file(config_path):
     # how many folds should we run in parallel for grid search
     grid_search_folds = config.getint("Tuning", "grid_search_folds")
 
-    # what is the objective function for the grid search?
-    grid_objective = config.get("Tuning", "objective")
-    if grid_objective not in SCORERS:
-        raise ValueError('Invalid grid objective function: {}'
-                         .format(grid_objective))
+    # what are the objective functions for the grid search?
+    grid_objectives_string = config.get("Tuning", "objective")
+    grid_objectives = yaml.load(_fix_json(grid_objectives_string))
+    if isinstance(grid_objectives, str):
+        grid_objectives_list = [grid_objectives]
+    elif isinstance(grid_objectives, list):
+        grid_objectives_list = grid_objectives
+    else:
+        raise ValueError("objective should be a string or "
+                         "list of objectives")
+
+    if not set(grid_objectives_list).issubset(set(SCORERS.keys())):
+        raise ValueError('Invalid grid objective function/s: {}'
+                         .format(list(grid_objectives)))
```

Comment:
You can just use `grid_objectives_list` here instead of creating a list again from `grid_objectives`?